main.js

```diff
@@ -1,15 +1,31 @@
-
 function makePyramid(){
+	
```

Comment:
Be very careful with whitespace. Inevitably it will lead to merge conflicts. You can change your settings in sublime to automatically trim the whitespace. Just enter this line into your custom settings.
`"trim_trailing_white_space_on_save": true,`

Also, a good way to visualize trailing whitespace is to add the Trailing Spaces package to your editor. It will highlight trailing whitespace in your code, making it really easy to stay on top of.